__tests__/pages/index.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+
+import Home from '../../pages/index';
```

Comment:
```suggestion
import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import Home from '../../pages/index';
```

Reply:
@axross oh I see, It was my intention. I wanted to separate import from Library and our files but I ok!